#### Import packages and retrieve data from file

In [13]:
import json
import pandas as pd
with open('myfile.json') as f:
    data = json.load(f)



Data Overview first 10 rows

In [12]:
df = pd.read_json('myfile.json')
print(df.head(10))

  FIPS Admin2                Province_State       Country_Region  \
0                                                    Afghanistan   
1                                                        Albania   
2                                                        Algeria   
3                                                        Andorra   
4                                                         Angola   
5                                                     Antarctica   
6                                            Antigua and Barbuda   
7                                                      Argentina   
8                                                        Armenia   
9              Australian Capital Territory            Australia   

           Last_Update       Lat      Long_  Confirmed  Deaths Recovered  \
0  2023-03-10 04:21:03  33.93911  67.709953     209451    7896             
1  2023-03-10 04:21:03   41.1533    20.1683     334457    3598             
2  2023-03-10 04:21:03 

last 10 rows

In [14]:
print(df.tail(10))

     FIPS Admin2 Province_State        Country_Region          Last_Update  \
4006                                          Uruguay  2023-03-10 04:21:03   
4007                                       Uzbekistan  2023-03-10 04:21:03   
4008                                          Vanuatu  2023-03-10 04:21:03   
4009                                        Venezuela  2023-03-10 04:21:03   
4010                                          Vietnam  2023-03-10 04:21:03   
4011                               West Bank and Gaza  2023-03-10 04:21:03   
4012                             Winter Olympics 2022  2023-03-10 04:21:03   
4013                                            Yemen  2023-03-10 04:21:03   
4014                                           Zambia  2023-03-10 04:21:03   
4015                                         Zimbabwe  2023-03-10 04:21:03   

             Lat       Long_  Confirmed  Deaths Recovered Active  \
4006    -32.5228    -55.7658    1034303    7617                    
4007 

Python version used

In [5]:
print(pd.__version__)

2.0.3


List number of columns and associated data types

In [15]:
print(df.info())

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 4016 entries, 0 to 4015
Data columns (total 14 columns):
 #   Column               Non-Null Count  Dtype 
---  ------               --------------  ----- 
 0   FIPS                 4016 non-null   object
 1   Admin2               4016 non-null   object
 2   Province_State       4016 non-null   object
 3   Country_Region       4016 non-null   object
 4   Last_Update          4016 non-null   object
 5   Lat                  4016 non-null   object
 6   Long_                4016 non-null   object
 7   Confirmed            4016 non-null   int64 
 8   Deaths               4016 non-null   int64 
 9   Recovered            4016 non-null   object
 10  Active               4016 non-null   object
 11  Combined_Key         4016 non-null   object
 12  Incident_Rate        4016 non-null   object
 13  Case_Fatality_Ratio  4016 non-null   object
dtypes: int64(2), object(12)
memory usage: 439.4+ KB
None


There are 4016 rows indexed from (0 to 4016). 

There are 14 columns.

there are no null values in the initial dataset.


Duplication Removal


In [16]:
print("Data set has duplicated rows? ", df.duplicated())

Data set has duplicated rows?  0       False
1       False
2       False
3       False
4       False
        ...  
4011    False
4012    False
4013    False
4014    False
4015    False
Length: 4016, dtype: bool


If dataset hadd duplicates, we can use the "drop_duplicates(inplace = True)" method

Find Correlations between each coloum in dataset

Separate all 14 columns 

In [24]:
c1 = pd.Series(df.loc(1))
#https://www.w3schools.com/python/pandas/pandas_correlations.asp

use python lib to visualize json data